### PR TITLE
Add docstring for Bio.NeuralNetwork

### PR DESCRIPTION
**Pull request by souravsingh**
Fixes a part of #1203

### CODE REVIEW

File: Bio/NeuralNetwork/Gene/Schema.py [hunk 1]
```diff
@@ -466,6 +466,13 @@ class SchemaFinder(object):
     """
     def __init__(self, num_schemas=100,
                  schema_finder=GeneticAlgorithmFinder()):
+        """Initialize the Schema Finder with its parameters.
+
+        Arguments:
+
+        o num_schemas -- the number of useful (positive fitness) schemas
```
Comment (peterjc) [inline Bio/NeuralNetwork/Gene/Schema.py]: No letter ``o`` for bullet points - please use valid RST markup, which you can check using e.g. http://rst.ninjs.org/

File: Bio/NeuralNetwork/Gene/Schema.py [hunk 2]
```diff
@@ -470,7 +470,7 @@ def __init__(self, num_schemas=100,
 
         Arguments:
 
-        o num_schemas -- the number of useful (positive fitness) schemas
+        * num_schemas -- the number of useful (positive fitness) schemas
```
Comment (peterjc) [inline Bio/NeuralNetwork/Gene/Schema.py]: Using ``*`` is valid RST, but for consistency with previous examples elsewhere in Biopython please use a minus sign.

Also, you must intent the next line by another two spaces as right now this is not valid RST.

Comment (souravsingh) [inline Bio/NeuralNetwork/Gene/Schema.py]: Should I make the fix for bulleting everywhere?

Comment (peterjc) [inline Bio/NeuralNetwork/Gene/Schema.py]: Yes please - GitHub must have only been showing me the latest changes.

**Comment by peterjc** [comment]
It seems you started this branch from a commit back in April, which explains why there are merge conflicts.

**Comment by peterjc** [comment]
This also explains why there were still letter ``o`` bullet points present, I fixed that a few weeks ago in 28d484055ede55fa66ff1051c5365552c2c66b3a which was a global search/replace.

**Comment by peterjc** [comment]
I believe I've manually applied your changes as https://github.com/biopython/biopython/commit/04948a43a55cb098df6e0d19e0626b010fe838fe - thanks!

Next time please make sure your repository is up to date before starting a new branch, eg if you've got ``upstream`` pointing at the official repository on GitHub try:

```
git checkout master
git fetch upstream
git rebase upstream/master
git checkout -b new_work
```

**Pull request closed without merging**